UsbCore.h

```diff
@@ -52,6 +52,8 @@ typedef MAX3421e<P15, P5> MAX3421E; // ESP8266 boards
 typedef MAX3421e<P5, P17> MAX3421E; // ESP32 boards
 #elif (defined(__AVR_ATmega644P__) || defined(__AVR_ATmega1284P__))
 typedef MAX3421e<Pb4, Pb3> MAX3421E; // Sanguino
+#elif defined(AM_PART_APOLLO3)
+typedef MAX3421e<P10, P9> MAX3421E; // Sparkfun Redboard Artemis All the pins
```

Comment:
No need to add this, as pin 10 and 9 are already the default i.e. remove these two lines.

Reply:
Thanks, I had put this in there in case I needed to change it later on. Removed